spec/features/researchers/coach_assignments_spec.rb

```diff
@@ -64,12 +64,11 @@
     expect(page).to have_text "Current Coach/Moderator: user2@example.com"
   end
 
-  it "doesn't allows for the assigning of no coach" do
+  it "doesn't allows for the assigning of coach if no group exists" do
```

Comment:
@Tag0Mag0 what do you think about writing a unit test instead on the Participant concern? Does this cover more ground as a feature spec than it would otherwise?